assembly-syntax/src/ast/procedure/name.rs

```diff
@@ -70,6 +70,14 @@ impl FromStr for QualifiedProcedureName {
     }
 }
 
+impl TryFrom<&str> for QualifiedProcedureName {
+    type Error = Report;
+
+    fn try_from(name: &str) -> Result<Self, Self::Error> {
+        Self::from_str(name)
+    }
+}
```

Comment:
Should we also add `TryFrom<String> for QualifiedProcedureName` for convenience?